{
  try {
    TestClass3.class.newInstance();
    fail("IllegalAccessException is not thrown.");
  }
 catch (  IllegalAccessException iae) {
  }
  try {
    TestClass1C.class.newInstance();
    fail("ExceptionInInitializerError should be thrown.");
  }
 catch (  java.lang.ExceptionInInitializerError ie) {
  }
}
